csharp/ql/src/change-notes/2025-04-10-uncontrolled-format-string.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The precision of the query `cs/uncontrolled-format-string` has been improved. Calls to `System.Text.CompositeFormat.Parse` are now considered a format like method call.
```

Comment:
cc @coadaflorin

Reply:
> The precision of the query `cs/uncontrolled-format-string` has been improved.

Something to explain the kind of improvements will make it easier to understand which way it goes, less FNs, or less FPs.

Reply:
@coadaflorin : The change note has been updated. Could you check and approve the PR in case the update is acceptable? Just for future reference - may I assume that the consumers of change notes, know about the abbreviations FPs, FNs and TPs or is better to spell them out?

Reply:
For simplicity let's spell them out. If someone doesn't know what a false positive is, at least they have a chance of finding something if they google it.